sign-up.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓 - 회원가입</title>
+    <link rel="icon" href="img/panda_favicon.png">
+    <link rel="stylesheet" href="/style/sign-up.css">
+    <link rel="stylesheet" href="/style/reset.css">
+    <link rel="stylesheet" href="/style/global.css">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"/>
+</head>
+<body>
+    <form>
+        <header>
+            <a href="/"><img class="logo" src="/img/logo_big.png" alt="판다마켓 로고"></a>
+        </header>
+
+        <div>
+            <label for="email">이메일</label>
+            <input id="email" type="email" name="email">
+        </div>
+        <div>
+            <label for="nickname">닉네임</label>
+            <input id="nickname" type="text" name="nickname">
+        </div>
+        <div class="password">
+            <label for="password">비밀번호</label>
+            <input id="password" type="password" name="password">
+            <img class="show-password" src="/img/visibility_off.png">
```

Comment:
❗️ 수정요청
해당 요소는 클릭 시 비밀번호가 보이고 안 보이게 만드는 요소가 될 것이므로 button요소로 작성하시는 것을 추천드려요.

```suggestion
            <button type='button'><img class="show-password" src="/img/visibility_off.png"></button>
```